app/interfaceactiontablehandler.py

```diff
@@ -368,17 +324,7 @@ async def get(self, request):
 
             get_job_roles = get_distinct_job_role_short()
         except ClientResponseError as ex:
-            if ex.status == 503:
-                logger.warn('Server is unavailable',
-                            client_ip=request['client_ip'])
-                flash(request, SERVICE_DOWN_MSG)
-                return aiohttp_jinja2.render_template('error503.html', request,
-                                                      {'include_nav': False})
-            else:
-                raise ex
-
-        except ClientResponseError as ex:
-            raise ex
+            client_response_error(ex, request)
```

Comment:
This (and all other instances) probably need to be `return ...` to return the constructed template in the event that the function does not throw.

Reply:
Done